drivers/storage/portworx/portworx.go

```diff
@@ -127,7 +127,9 @@ func (p *portworx) Validate(cluster *corev1.StorageCluster) error {
 		util.UpdateStorageClusterCondition(cluster, condition)
 	}
 
-	podSpec, err := p.GetStoragePodSpec(cluster, "")
+	preFltCluster := GetPreFlightStorageCluster(cluster)
```

Comment:
It is fine to do it this way for now. In future we would ideally want to send the exact same parameters to the preflight that we would send to the actual px/oci-monitor pod. That way it can run all the prechecks.